<?xml version="1.0"?>
<launch>=
    <include file="$(find gopher_ros_clearcore)/launch/chest.launch"/>
    <include file="$(find unity_ros)/launch/endpoint.launch"/>

    <node pkg="oculus" type="oculus_feedback.py" name="oculus_feedback" output="screen"/>
    <node pkg="oculus" type="oculus_chest_mapping.py" name="oculus_chest_mapping" output="screen"/>
    <node pkg="oculus" type="oculus_mobile_base_mapping.py" name="oculus_mobilebase_mapping" output="screen"/>
    <node pkg="oculus" type="oculus_kinova_mapping.py" name="oculus_kinova_mapping" output="screen"/>

</launch>
